requestsEx.py

```diff
@@ -0,0 +1,25 @@
+import requests
+import sys
+
+
+def top_repository_from_github(programming_language: str, number: int):
+    """
+
+    The function receives a language and a number, and brings from the GitHub the number of repositories according
+    to the number and language received, arranged according to the number of stars in descending order.
+    :param programming_language: The language in which you want the repositories.
+    :param number: The language in which you want the repositories.
+    """
+    headers = {'Accept': 'application/vnd.github.v3+json'}
+    username = sys.argv[1]
+    token = sys.argv[2]
+    response = requests.get('https://api.github.com/search/repositories?q=language:' + programming_language +
```

Comment:
Use double quotes on strings - 
https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python